index.d.ts

```diff
@@ -1,3 +1,27 @@
+interface Options {
```

Comment:
`Options` needs to be in a namespace, so it can be imported by the user: https://github.com/sindresorhus/typescript-definition-style-guide#documentation